<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAcquisitionForm</class>
 <widget class="QWidget" name="DataAcquisitionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="dataFilePrefix">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>36</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="offsetsButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Offsets?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="mode">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abortDAQ">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>153</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abort DAQ</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataFileDirectoryLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Data File Directory</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="state">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Idle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dataFileDirectoryButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>8</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Directory</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dataFileTimestamp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>36</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Append Timestamp?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="trigger">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>120</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="collectImages">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Collect Images</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="duration">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="dataFilePrefixLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Data File Prefix</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataFileDirectory">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dataAcquisitionModeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondsLabel">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="housingTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>297</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Housing Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="housingTemperature">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>297</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="housingHumidityLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>322</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Housing Humidity</string>
   </property>
  </widget>
  <widget class="QLabel" name="housingHumidity">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>322</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="fingerTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>345</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Finger Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="fingerTemperature">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>345</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="stopTrigger">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Trigger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initTrigger">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialise Trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="biasLastRefreshedLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bias last refreshed at</string>
   </property>
  </widget>
  <widget class="QLabel" name="biasLastRefreshed">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="repeatCount">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="timesLabel">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>60</y>
     <width>64</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>times,  every</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="repeatInterval">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="secondsLabel_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="detectorTemperature">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="detectorTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Detector Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataAcquisitionModeLabel_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="estimatedDuration">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hhh:mm:ss.ss&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="estimatedDurationLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimated DAQ Duration</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="imageProgressBar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="progressBarLabel">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>210</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="biasCurrent">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>260</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="biasCurrentLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bias Current</string>
   </property>
  </widget>
  <widget class="QLabel" name="dewPoint">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="dewPointLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dew Point</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="logFileDirectory">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>324</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="logFilePrefix">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logFileDirectoryButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>322</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="logFilePrefixLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Log File Prefix</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="logFileDirectoryLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>324</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Log File Directory</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="logFileTimestamp">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Append Timestamp?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="loggingEnabled">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Logging?</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="progressBarLabel_2">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DAQ Progress</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
